xbmc/cores/VideoPlayer/Process/VideoBuffer.cpp

```diff
@@ -318,14 +318,21 @@ void CVideoBufferPoolSysMem::Return(int id)
   m_free.push_back(id);
 }
 
-void CVideoBufferPoolSysMem::Configure(AVPixelFormat format, int width, int height)
+void CVideoBufferPoolSysMem::Configure(AVPixelFormat format, int width, int height, int aligned_width, int aligned_height, int size)
```

Comment:
Unfortunately that doesn't work for AddonVideoCodec.
It only provides the buffer size and not the width/height (or stride) when Get is called.
You only see the width/height (and stride) when the decoded picture is returned.